ts/sagas/startup.ts

```diff
@@ -322,6 +325,11 @@ export function* initializeApplicationSaga(): Generator<Effect, void, any> {
       // check if the user expressed preference about mixpanel, if not ask for it
       yield call(askMixpanelOptIn);
 
+      if (servicesRedesignEnabled) {
+        // TODO the saga should be called even for already loggedIn users without the preference set
```

Comment:
Here you're not already in the user logged in case?

Reply:
Yes right it was a CP&paste 😄 

```suggestion
        // TODO evaluate a refactor of the saga as part of https://pagopa.atlassian.net/browse/IARS-25
```